inbm/dispatcher-agent/dispatcher/aota/application_command.py

```diff
@@ -93,7 +93,7 @@ def _reboot(self, cmd: str) -> None:
             logger.debug(f" Application {self.resource} installed. Rebooting...")
             self._dispatcher_callbacks.broker_core.telemetry('Rebooting...')
             (output, err, code) = PseudoShellRunner.run(cmd)
-            if code != 0:
+            if code != 0 and code != -15:
```

Comment:
what is code -15?

Reply:
i believe its the SIGTERM value.

Reply:
Could we use a constant for this instead with what it actually is.  We shouldn't use magic numbers.

Reply:
PR updated. The constants for magic numbers are added and used.